lab/requirements.md

```diff
@@ -0,0 +1,53 @@
+# OPI Lab Requirements
+
+## Access
+
+- Lab staff can access to all machines.
+- OPI members should access only to their own machines.
+- Remote access to a machine requires private VPN specific to the machine.
+- OPI members cannot access to a machine from another one.
```

Comment:
I don't follow, you're saying we're not allowing east-west connections? Or, if I have, say, `machine a` reserved, someone on `machine b` can't access `machine a`? That's why we need this lab to be multi-tenant at the network level.

Reply:
Yes that's what I mean, no east-west. I think connection to the traffic generator is enough to test xPU features.

Reply:
If we use a bare metal provisioning system which can separate resources with some as basic as VLANs we can solve this issue. Obviously there are other network segmentation protocols we could use as well.

Reply:
We have to use hardware in a limited fashion. While I understand the need for multi-node setups that can be a security liability. VLAN based segmentation doesn't guarantee security.

Reply:
Don't focus on VLAN, it was an example. My point is we need to enable multi-tenancy here.